src/istex-scroll.js

```diff
@@ -0,0 +1,25 @@
+import fetch from 'omni-fetch';
```

Comment:
This is not bound to the scroll mechanism.
It only fetches an URL, and convert the result to JSON.
Oh sorry, there is a test on `json.total`.

The name is confusing to me: I thought that the loop was here, whereas it is in ISTEXSearch: the original query is converted to a series of queries!